dvc/fs/gdrive.py

```diff
@@ -127,6 +126,9 @@ def __init__(self, repo, config):
         self._client_id = config.get("gdrive_client_id")
         self._client_secret = config.get("gdrive_client_secret")
         self._validate_config()
+        self._gdrive_service_credentials_path = tmp_fname(
+            os.path.join(self.repo.tmp_dir, "")
```

Comment:
Let's use some prefix to identify it as a one belonging to gdrive?

Reply:
we clean this file, so it doesn't exist after the auth call (also this is what we do for the normal files [non service accounts])